ts/Series/FlowMap/FlowMapSeries.ts

```diff
@@ -0,0 +1,834 @@
+/* *
+ *
+ *  (c) 2010-2022 Askel Eirik Johansson, Piotr Madej
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *  Imports
+ *
+ * */
+
+import type FlowMapSeriesOptions from './FlowMapSeriesOptions';
+import type { MarkerEndOptions } from './FlowMapPointOptions';
+import type SVGPath from '../../Core/Renderer/SVG/SVGPath';
+import FlowMapPoint from './FlowMapPoint.js';
+import MapSeries from '../Map/MapSeries.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+
+const {
```

Comment:
There is a new way of series import, that we should probably follow, e.g.: https://github.com/highcharts/highcharts/pull/17618/files#diff-dfdcaa3c1114b26145bb94ad79d0d629864c59984464a77fe50398140163097c

Reply:
@magdalena-gut after checking it, it turned out that the current way of importing it is correct. The presented method is rather a workaround in cases where it is not correctly interpreted by TypeScript and when, for example, it starts complaining about the type scope when generating native Declarations from our sources. In my opinion, the current way is more readable for this example